packages/api/src/external/commonwell/patient/cw-patient-data/shared.ts

```diff
@@ -19,6 +19,21 @@ export interface CwPatientDataCreatePartial extends BaseDomainCreate {
   data: Partial<CwData>;
 }
 
+export type CwDataV2 = {
+  links: CwLinkV2[];
+  linkDemographicsHistory?: LinkDemographicsHistory;
+};
+
+export interface CwPatientDataCreateV2 extends BaseDomainCreate {
+  cxId: string;
+  data: CwDataV2;
+}
+
+export interface CwPatientDataCreatePartialV2 extends BaseDomainCreate {
+  cxId: string;
+  data: Partial<CwDataV2>;
+}
+
```

Comment:
this is partialed to support an update operation, right? Couid / shouldn't we call this CwPatientDataUpdateV2 instead?